opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -91,9 +91,64 @@
 logger = logging.getLogger(__name__)
 
 
-TELEMETRY_SDK_LANGUAGE = "telemetry.sdk.language"
+CLOUD_PROVIDER = "cloud.provider"
```

Comment:
Why do we have all these variables if we are not using them?

Reply:
These are the standard resource attributes, having these in the SDK allow code outside to use them correctly and avoid typos. 

Node.js  
https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-resources/src/constants.ts